index.bs

```diff
@@ -724,6 +725,61 @@ When an {{XRSession}} |session| receives updated [=viewer=] state from the [=/XR
 
 </div>
 
+<div class="example">
+Depending on the device being used, callbacks supplied to {{Window}} {{Window/requestAnimationFrame()}} may not be processed while and [=immersive session=] is active. (For instance, on a mobile or standalone device where the immersive content completely obscures the HTML document.) As such, developers should not rely on {{Window}} {{Window/requestAnimationFrame()}} callbacks to schedule {{XRSession}} {{XRSession/requestAnimationFrame()}} callbacks and visa-versa, even if they share the same rendering logic, or the applications animation loop may not run properly on all platforms. A more effective pattern for applications that wish to transition between these two types of animation loops is demonstrated below:
```

Comment:
Some browsers support "warnings" in devtools. Should we have a non-normative note suggesting browsers show a warning for `session.rAF()` called within `window.rAF()` or vice versa?

Reply:
I've wondered about it, but I'm not fully convinced that there's no legitimate use cases for something like that? I don't know, would have to give it some more thought.

Reply:
Heck, could we do something more drastic and reject `session.rAF()` if invoked from `window.rAF()`?  If that's too much, maybe we reject it from there if it's never been hooked before?

Thinking about the consequences of such a choice... in the future, some layers won't ever require processing a rAF (for example a hardware-backed video projection layer).... so what would happen when the video ends and the developer wants to start rendering their own content with a traditional layer type... when would we expect them to hook the `session.rAF()`?  I do worry they might try to use the `window.rAF()` if not prevented from doing so...

Reply:
> Heck, could we do something more drastic and reject session.rAF() if invoked from window.rAF()? 

I considered suggesting this but it felt a bit too drastic to me. I'm not against it though.